### ISSUE TITLE
Nylas N1 support

### ISSUE THREAD

**Issue by bil-elmoussaoui** [bil-elmoussaoui/Hardcode-Tray#153]
Nylas is another Electron app

**Comment by wa4557** [comment]
key for the icons is:

    ['files']['static']['files']['images']['files']['toolbar']['files']

then there are a lot of files there:
    u'toolbar-popout@2x.png',
    u'toolbar-star-selected@1x.png',
    u'toolbar-markasread@1x.png',
    u'toolbar-movetofolder@2x.png',
    u'toolbar-folder@2x.png',
    u'toolbar-tag@1x.png',
    u'toolbar-reply-all@2x.png',
    u'toolbar-attach@2x.png',
    u'toolbar-icon-toggle-pane@1x.png',
    u'toolbar-style@2x.png',
    u'toolbar-person-sidebar@1x.png',
    u'toolbar-spam@2x.png',
    u'toolbar-trash@1x.png',
    u'toolbar-forward@2x.png',
    u'toolbar-more@2x.png',
    u'toolbar-star-selected@2x.png',
    u'toolbar-send@2x.png',
    u'toolbar-templates@2x.png',
    u'toolbar-archive@1x.png',
    u'toolbar-trash@2x.png',
    u'toolbar-archive@2x.png',
    u'searchloupe@2x.png',
    u'toolbar-icon-toggle-pane@2x.png',
    u'toolbar-markasunread@2x.png',
    u'toolbar-compose@2x.png',
    u'toolbar-star@2x.png',
    u'toolbar-reply@2x.png',
    u'toolbar-compose@1x.png',
    u'toolbar-chevron@2x.png',
    u'toolbar-dropdown-chevron@2x.png',
    u'toolbar-snooze@1x.png',
    u'toolbar-markasread@2x.png',
    u'toolbar-movetofolder@1x.png',
    u'toolbar-person-sidebar@2x.png',
    u'toolbar-tag@2x.png',
    u'toolbar-spam@1x.png',
    u'toolbar-snooze@2x.png',
    u'toolbar-markasunread@1x.png',
    u'toolbar-star@1x.png']

**Comment by bil-elmoussaoui** [comment]
Are you sure? I found 3 icons
```json
"system-tray": {
          "files": {
            "assets": {
              "files": {
                "darwin": {
                  "files": {
                    "MenuItem-Inbox-Full-NewItems@1x.png": {
                      "offset": "29410001",
                      "size": 14745
                    },
                    "MenuItem-Inbox-Full-NewItems@2x.png": {
                      "offset": "29424746",
                      "size": 14986
                    },
                    "MenuItem-Inbox-Full@1x.png": {
                      "offset": "29439732",
                      "size": 14753
                    },
                    "MenuItem-Inbox-Full@2x.png": {
                      "offset": "29454485",
                      "size": 14985
                    },
                    "MenuItem-Inbox-Zero@1x.png": {
                      "offset": "29469470",
                      "size": 14770
                    },
                    "MenuItem-Inbox-Zero@2x.png": {
                      "offset": "29484240",
                      "size": 15045
                    }
                  }
                },
                "linux": {
                  "files": {
                    "MenuItem-Inbox-Full-NewItems@2x.png": {
                      "offset": "29499285",
                      "size": 19245
                    },
                    "MenuItem-Inbox-Full@2x.png": {
                      "offset": "29518530",
                      "size": 19031
                    },
                    "MenuItem-Inbox-Zero@2x.png": {
                      "offset": "29537561",
                      "size": 19031
                    }
                  }
                },
                "win32": {
                  "files": {
                    "MenuItem-Inbox-Full-NewItems@2x.png": {
                      "offset": "29556592",
                      "size": 21047
                    },
                    "MenuItem-Inbox-Full@2x.png": {
                      "offset": "29577639",
                      "size": 20471
                    },
                    "MenuItem-Inbox-Zero@2x.png": {
                      "offset": "29598110",
                      "size": 20471
                    }
                  }
                }
              }
            },
            "lib": {
              "files": {
                "main.es6": {
                  "offset": "29618581",
                  "size": 254
                },
                "main.js": {
                  "offset": "9848843",
                  "size": 1540
                },
                "system-tray-icon-store.es6": {
                  "offset": "29618835",
                  "size": 2388
                },
                "system-tray-icon-store.js": {
                  "offset": "9850383",
                  "size": 10455
                }
              }
            },
            "package.json": {
              "offset": "29621223",
              "size": 255
            },
            "spec": {
              "files": {
                "system-tray-icon-store-spec.es6": {
                  "offset": "29621478",
                  "size": 2776
                },
                "system-tray-icon-store-spec.js": {
                  "offset": "29624254",
                  "size": 11524
                }
              }
            }
          }
        }
```

**Comment by wa4557** [comment]
oh good I looked at the wrong icons... Actually the `electron` script should work for them as well

**Comment by bil-elmoussaoui** [comment]
But i can't get the main key of `system-tray`

**Comment by wa4557** [comment]
Yeah me neither...

**Comment by bil-elmoussaoui** [comment]
fixed in 4e384fcec074f1b126151c7630510cb8d3fa66db

**Comment by bil-elmoussaoui** [comment]
@wa4557 if you still have issues finding the main key to the tray icons, what i do is to extract the asar file to a folder using `asar` npm package. and look for the icons using my default files manager and once i have the path to the icons i add `files/` before each folder name and it works: p
